common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -87,18 +87,43 @@
                     frontId.getOrElse(pageId)
                     )) { case (isFirstContainer, id) =>
 
-                    @if(mvt.ABNewHeaderVariant.isParticipating && isFirstContainer) {
+                    @if((mvt.ABNewHeaderVariant.isParticipating || mvt.ABNewHeaderVariantTwo.isParticipating) && isFirstContainer) {
 
                         <ul class="tertiary-navigation mobile-only">
 
-                            @NewNavigation.TertiaryNav.getTertiaryNavLinks(id).map { tertiaryLink =>
-
-                                <li class="tertiary-navigation__item">
-                                    <a class="tertiary-navigation__link" href="@LinkTo(tertiaryLink.url)" data-link-name="nav2 : tertiary : @tertiaryLink.name">
-                                        <span class="tertiary-navigation__text">@Html(tertiaryLink.name)</span>
-                                        <i class="tertiary-navigation__icon rounded-icon"></i>
-                                    </a>
-                                </li>
+                            @* TODO: we should make sure that sport, culture, and life have tertiary links *@
+                            @* TODO: should this only happen on the second variant?*@
+                            @defining(Edition(request)) { edition: Edition =>
+                                @println(edition.networkFrontId)
```

Comment:
Nope! Thanks for that. I have removed it in https://github.com/guardian/frontend/pull/14676/commits/72c15a48fb61bb22840bcb9dd44fb537d26d97d6